### PR TITLE
fix: use PreciseDate instead of Date for min read timestamp

### PR DESCRIPTION
**Pull request by agasheesh**
Without this fix, TimestampBounds defaults min read timestamp to empty google timestamp proto, which correspond to min unix time i.e., 1970:01:01-T:00:00:00Z.

### CODE REVIEW

File: system-test/spanner.ts [hunk 1]
```diff
@@ -3457,7 +3458,7 @@ describe('Spanner', () => {
         const query = 'SELECT * FROM TxnTable';
 
         const options = {
-          minReadTimestamp: new Date(),
+          minReadTimestamp: new PreciseDate(),
```
Comment (skuruppu) [inline system-test/spanner.ts]: I couldn't really tell from the [docs](https://github.com/googleapis/nodejs-precise-date) what PreciseDate() defaults to for the empty ctor. Is it some value or is it just an empty object?

Comment (agasheesh) [reply system-test/spanner.ts]: PreciseDate() will call ctor for super() i.e., Date() which returns Now() timestamp. TimestampBounds object however expects an object of PreciseDate and will convert it correctly to a string representation.

Comment (agasheesh) [reply system-test/spanner.ts]: https://screenshot.googleplex.com/X54iBrh6WFQ

Comment (agasheesh) [reply system-test/spanner.ts]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

Comment (skuruppu) [reply system-test/spanner.ts]: Ah got it, thanks for the pointer @agasheesh.

**Pull request merged**